spec/models/post_mover_spec.rb

```diff
@@ -120,7 +120,7 @@ def add_moderator_post_to(topic, post_type)
             .to  change { p2.reload.topic_id }
             .and change { p2.post_number }
             .and change { p3.reload.raw }
-            .and change { p3.baked_version }.to nil
+            .and change { p2.baked_version || p3.baked_version }.to nil
```

Comment:
This is a little confusing to read because we're try to test attributes of two different objects in a single `change` block. I'll suggest changing to something like the following:

```suggestion
            .and change { p3.baked_version }.to nil
            .and change { p2.baked_version }.to nil 
```

Reply:
I already tried it. Your code suggestion won't work since it means **`nil.and`**` change { p2.baked_version }.to nil `

Reply:
I'm not sure I follow. Which `baked_version` will change here?

Reply:
Both of them

Reply:
```suggestion
            .and change { p2.baked_version }.to(nil)
            .and change { p3.baked_version }.to(nil)
```

O it is a syntax problem. Try the above instead :)